src/gitProjectManager.js

```diff
@@ -48,9 +51,17 @@ class GitProjectManager {
         });
 
         return this.repoList.map(repo => {
+            let description;
+            if (this.config.checkRemoteOrigin && this.config.displayProjectPath) {
+                description = `${GLOBE} ${repo.repo} ${FOLDER} ${repo.dir}`;
```

Comment:
Could we shorten repo.dir by using `~` in place of process.env.HOME?

Reply:
Done :ok_hand: